Admin/PageAdmin.php

```diff
@@ -290,7 +290,10 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                     'Sonata\PageBundle\Form\Type\PageTypeChoiceType' :
                     'sonata_page_type_choice',
             ))
-//            ->add('type', null, array('field_type' => 'Sonata\PageBundle\Form\Type\PageTypeChoiceType'))
+/*
+ * NEXT_MAJOR
+ *          ->add('type', null, array('field_type' => 'Sonata\PageBundle\Form\Type\PageTypeChoiceType'))
+ */
```

Comment:
I was gonna ask why this is not indented but it looks like it was like this in the first place...